ethereumkit/src/main/java/org/p2p/ethereumkit/external/model/ERC20Tokens.kt

```diff
@@ -100,6 +91,7 @@ enum class ERC20Tokens(
     companion object {
 
         const val ETH_DECIMALS = 18
+        const val SOL_TOKEN_URL = "https://assets.coingecko.com/coins/images/4128/large/solana.png?1640133422"
```

Comment:
I don't think it's connected with ERC20 tokens. Maybe Const class is better?

Reply:
it is used only for bridges
but we can move it to Constants

but anyway it is partially connected =)